### PR TITLE
Image list as grid

### PR DESCRIPTION
**Pull request by wrobelmichal**

### CODE REVIEW

**Review by wilmhit — CHANGES_REQUESTED** [review]
I'm getting stretched images. Also could be still a little bit smaller (I am thinking of something like YouTube main page thumbnails), but that's debatable. Otherwise, it really looks good.
image

File: easywarehouse/static/images.css [hunk 1]
```diff
@@ -1,4 +1,24 @@
 img{
-    height: 50vh;
+    max-height: 50vh;
```
Comment (wilmhit) [inline easywarehouse/static/images.css:2]: Please don't use vh/vw for something like this. Pixels are fine (they are not really pixels).

File: easywarehouse/static/images.css [hunk 2]
```diff
@@ -1,4 +1,24 @@
...
+    margin: 5px;
+    display: flex;
+    justify-content: center;
+}
+
+ul li img{
+    margin: 5px;
+    display: flex;
+    justify-content: center;
+    max-width: 30vh;
```
Comment (wilmhit) [inline easywarehouse/static/images.css:23]: Here too.

File: easywarehouse/templates/employee/images/list.html [hunk 3]
```diff
@@ -2,10 +2,11 @@
 {% load static %}
 
 {% block content %}
-    <link rel="stylesheet" type="text/css" href="{% static 'list.css' %}">
+<link rel="stylesheet" type="text/css" href="{% static 'list.css' %}" xmlns="http://www.w3.org/1999/html">
```
Comment (wilmhit) [inline easywarehouse/templates/employee/images/list.html:5]: I don't think this is necessary any more. IE days are over. Correct me if I'm wrong.

**Review by wilmhit — CHANGES_REQUESTED** [review]
We discussed using `.action-bar` style earlier today. Image listing page should use it just for consistency, but if You want to do it in another pull request that's fine

File: easywarehouse/templates/employee/images/list.html (hunk 3 above)
Comment (wrobelmichal) [reply easywarehouse/templates/employee/images/list.html:5]: I didn't write this attribute.

**Comment by wilmhit** [comment]
Should I wait for action bar in this pr?

**Comment by wrobelmichal** [comment]
Action bar will be in next pr

**Review by wilmhit — APPROVED** [review]

**Pull request merged**